cpp/src/arrow/flight/integration_tests/test_integration_client.cc

```diff
@@ -94,7 +94,7 @@ Status ConsumeFlightLocation(
     FlightClient* read_client, const Ticket& ticket,
     const std::vector<std::shared_ptr<RecordBatch>>& retrieved_data) {
   std::unique_ptr<FlightStreamReader> stream;
-  RETURN_NOT_OK(read_client->DoGet(ticket, &stream));
+  RETURN_NOT_OK(read_client->DoGet(ticket).Value(&stream));
```

Comment:
use ARROW_ASSIGN_OR_RAISE

Reply:
Did you mean to resolve this?

Reply:
Sorry forgot to push that. GLib code is still wrong, I'll fix that later